src/metadata/parse.rs

```diff
@@ -149,6 +149,39 @@ pub fn parse_editor(val: &str) -> Result<Editor, &'static str> {
     Ok(editor)
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct EditorTerm {
+    pub singular: String,
+    pub plural: String,
+}
+
+impl EditorTerm {
+    fn new(singular: String, plural: String) -> Self {
+        EditorTerm { singular, plural }
+    }
+}
+
+impl Default for EditorTerm {
+    fn default() -> Self {
+        EditorTerm::new("Editor".to_owned(), "Editors".to_owned())
+    }
+}
+
+pub fn parse_editor_term(val: &str) -> Result<EditorTerm, &'static str> {
+    // <editor-term> := <singular-term> "," <plural-term>
+
+    let pieces = val
+        .split(",")
+        .map(|piece| piece.trim())
+        .collect::<Vec<&str>>();
+
+    if pieces.len() == 2 {
+        Ok(EditorTerm::new(pieces[0].to_owned(), pieces[1].to_owned()))
+    } else {
+        Err("wrong format")
```

Comment:
May be worth to be a bit more descriptive in the error message (even if it's just "wrong editor term format"). When the errors are bubbled up the information of where they came from gets lost...